### DISCUSSION TITLE
FluentUI Examples

### DISCUSSION THREAD

**Discussion by Gareth064** [microsoft/fluentui-blazor#941] [Show and tell]
Hey all, is there a list of GitHub repos or links to web apps which have used FluentUI?
If not, should we start one so new people to the framework can see what others have done, to get inspiration.

**Comment by vnbaaij** [comment]
I don't know of such a list. If you want, you can start one. Maybe here?

Good place to start: https://github.com/microsoft/fluentui-blazor/network/dependents
